zarnitsa/nlp/DataAugmenterNLP.py

```diff
@@ -8,9 +8,9 @@
 import pandas as pd
 import nlpaug.augmenter.word as naw
 
-from sklearn.model_selection import train_test_split
+from sklearn.model_selection import train_test_split as splitting
```

Comment:
I think that `train_test_split` is quite popular and we shouldn't bring ambiguity with custom name here, but it's up to you